widgets.html

```diff
@@ -6,8 +6,22 @@
 
 <link href="css/gallery.css" rel="stylesheet">
 <link href="css/syntax.css" rel="stylesheet">
-<script src="https://unpkg.com/jupyter-js-widgets@~2.1.4/dist/embed.js"></script>
-
+<script
+        src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"
+        integrity="sha256-Ae2Vz/4ePdIu6ZyI/5ZGsYnb+m0JlOmKPjt6XZ9JJkA="
+        crossorigin="anonymous">
+</script>
+<script src="https://unpkg.com/@jupyter-widgets/html-manager@*/dist/embed-amd.js" crossorigin="anonymous"></script>
+<script>
+  // Configure requirejs
+  if (window.require) {
+    window.requirejs.onError = function(e) {
+      console.log(e);
+      // hack! Remove the anonymous require call from https://unpkg.com/@jupyter-widgets/html-manager@0.10.2/dist/embed-amd.js
+      // window.require(['@jupyter-widgets/html-manager/dist/libembed-amd'], ...
```

Comment:
These lines are not needed, right? Do we need to override the error function at all?

Reply:
As described in comment we need to resolve the issue with `anonymous require call from https://unpkg.com/@jupyter-widgets/html-manager@0.10.2/dist/embed-amd.js`.
Otherwise if you remove this `onerror` overridden method the requirejs will throw an error:
```
Error: Mismatched anonymous define() module: [object Object]
http://requirejs.org/docs/errors.html#mismatch
    at makeError (require.min.js:1)
    at v (require.min.js:1)
    at Object.o [as require] (require.min.js:1)
    at requirejs (require.min.js:1)
    at embed-amd.js:274986
    at Promise (<anonymous>)
    at requirePromise (embed-amd.js:274980)
    at Object.renderWidgets (embed-amd.js:275006)
    at embed-amd.js:281516
```
and stop working. So the examples won't work as their widgets won't be loaded by requirejs.

Reply:
@jasongrout 
After debugging the issue of `Mismatched anonymous define() module` it turned out that it's caused by the anonymous module inside the `https://buttons.github.io/buttons.js`. After adding the `requirejs` to `http://jupyter.org/widgets` the `buttons.js` script runs the 
```
if typeof define is "function" and define.amd
  define [], { render: render }
```
which causes the error to be thrown. It has nothing to do with the `ipywidgets` v7.

Reply:
Huh. This is part of the reason why we created an html manager that did *not* use require - because introducing require on a page changes how the js on the page works.

Reply:
But still, the example widgets are using the amd modules and depend on RequireJS.
So using the non amd version of `ipywidgets` will not resolve the issue here.

Reply:
afaik the simplest fix would be to not include buttons.js.
it's used to create the github and twitter buttons in the footer.
these buttons could be replaced with simple html.

Reply:
I would be :+1: on not including this js at all.

Reply:
I've got an answer from the lib author. His solution is to load the lib via the RequireJS. But this would force us to include the RequireJS on other pages as well. I'll provide the non js solution for the `Follow` button though.